recipes/libavif/all/test_package/test_package.c

```diff
@@ -0,0 +1,12 @@
+#include <avif/avif.h>
+#include <stddef.h>
+
+int main(int argc, char const* argv[])
```

Comment:
The standard prototype of the main() function does not have the "const":

`int main(int argc, char* argv[])`

See https://en.cppreference.com/w/c/language/main_function

Also, there is an alternative prototype that you can use here to avoid the `(void)argc` and `(void)argv` casts.

`int main(void)`